### ISSUE TITLE
How to use an optimizer in TFSharp?

### ISSUE THREAD

**Issue by ghulammustufa31** [migueldeicaza/TensorFlowSharp#314]
How can we use an optimizer, such as Gradient Descent or Adam optimizer?

**Comment by JohnyWaster** [comment]
I think it should be like this: https://stackoverflow.com/questions/49291781/how-to-train-model-with-tensorflowsharp
